lib/src/main/java/com/thenewboston/api/bankapi/datasource/BankDataSource.kt

```diff
@@ -125,4 +130,39 @@ class BankDataSource @Inject constructor(private val networkClient: NetworkClien
             else -> Outcome.Success(blocks)
         }
     }
+
+    suspend fun updateAccount(
+        accountNumber: String,
+        trustLevel: Double,
+        nodeIdentifier: String,
+        signature: String
+    ): Outcome<Account> = makeApiCall(
+        call = { doUpdateAccount(accountNumber, trustLevel, nodeIdentifier, signature) },
+        errorMessage = "Could not update trust level of given account"
+    )
+
+    private suspend fun doUpdateAccount(
+        accountNumber: String,
+        trustLevel: Double,
+        nodeIdentifier: String,
+        signature: String
+    ): Outcome<Account> {
+        val patchAccountUrl = "${Endpoints.ACCOUNTS_ENDPOINT}/$accountNumber"
```

Comment:
Can we add this to BankApiConstants class to store all the constants for this API?